includes/class-simple-local-avatars.php

```diff
@@ -407,7 +407,7 @@ class="button item-delete submitdelete deletion"
 					}
 
 					foreach ( $this->avatar_ratings as $key => $rating ) :
-						echo "\n\t<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( $key ) . "' " . checked( $profileuser->simple_local_avatar_rating, $key, false ) . "/> $rating</label><br />";
+						echo "\n\t<label><input type='radio' name='simple_local_avatar_rating' value='" . esc_attr( $key ) . "' " . checked( $profileuser->simple_local_avatar_rating, $key, false ) . "/>" . __($rating) . "</label><br />";
```

Comment:
@ActuallyConnor Thank you for raising a PR!

This looks good and fixes the issue but as per standards, the plain strings should always be directly passed as a string literal in the translation function, not as a variable. Read more [here](https://developer.wordpress.org/reference/functions/__/#comment-540).

> This seems to work, but it will interfere in automatic parsing of your plugin/theme’s files for translation.

Moreover, we are already translating strings.

https://github.com/10up/simple-local-avatars/blob/86ce042ae571583986d0c439675fc3f9be5be04a/includes/class-simple-local-avatars.php#L15-L20

But these are loaded prior to the user object being initialized as mentioned in #88. That needs to be fixed in a standard way.

Reply:
Understood, I can work on fixing that to work in a standard way